kingfisher_scrapy/spiders/tender_base.py

```diff
@@ -0,0 +1,44 @@
+from math import ceil
+
+import scrapy
+
+from kingfisher_scrapy.base_spider import IndexSpider
+from kingfisher_scrapy.util import parameters
+
+
+class TenderBase(IndexSpider):
+    """
+    Domain
+      TenderBase
+    API documentation
+      http://www.tenderbase.eu/releases/
+    """
+    name = 'tender_base'
```

Comment:
```suggestion
    name = 'europe_ted_tender_base'
```
And also rename the class and file name accordingly